oak-api/src/main/java/org/apache/jackrabbit/oak/api/StrictPathRestriction.java

```diff
@@ -0,0 +1,17 @@
+package org.apache.jackrabbit.oak.api;
+
+public enum StrictPathRestriction {
+    ENABLE("enable"),
+    WARN("warn"),
+    DISABLE("disable");
+    private String stringValue;
+
+    StrictPathRestriction(String pathRestriction) {
+        stringValue = pathRestriction;
+    }
+
+    public static StrictPathRestriction stringToEnum(String strictPathRestrictionInString) {
+        return StrictPathRestriction.valueOf(strictPathRestrictionInString.toUpperCase());
+    }
```

Comment:
I understand lowercase is a bit easier to read... but sometimes it is uppercase (setStrictPathRestriction below), sometimes lowercase (here). I think it's better if it's consistent (either upper or lowercase). I would be fine if it's consistently uppercase, if that simplifies the code (I guess it does). In which case this method wouldn't be needed.